readthedocs/subscriptions/utils.py

```diff
@@ -1,27 +1,29 @@
 """Utilities to interact with subscriptions and stripe."""
 
-import structlog
-
 import stripe
+import structlog
+from djstripe.models import Customer as StripeCustomer
```

Comment:
Does it make sense to use this as following instead?

```suggestion
import stripe
from djstripe.models as djstripe

stripe.Customer(...)
djstripe.Customer(...)
```

This way it's pretty clear where the model comes from.

Reply:
Yeah, wasn't sure the best way to scope the models, I like your suggestion :+1: